scripts/bundleSizeConfig.js

```diff
@@ -2,5 +2,5 @@ module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
   MIN_SIZE: 654,
-  MAX_SIZE: 715,
+  MAX_SIZE: 788,
```

Comment:
This is a much bigger bump than expected - I'll explain why in the package lock file!